pkgs/cuda-packages/setupCudaHook/setup-cuda-hook.sh

```diff
@@ -167,8 +167,8 @@ disallowNVCCHostCompilerLinkLeakage() {
     while IFS= read -r -d ':' rpath; do
       for forbiddenRPATH in "${cudaForbiddenRPATHs[@]}"; do
         if [[ $rpath == "$forbiddenRPATH"* ]]; then
-          echo "ERROR: forbidden path $forbiddenRPATH exists in RPATH of $libpath"
-          return 1
+          nixErrorLog "ERROR: forbidden path $forbiddenRPATH exists in RPATH of $libpath"
+          # return 1 # Don't error because we don't have a mechanism to fix this at the moment.
```

Comment:
Add a clear note here that this is a differences from what's in upstream.

Reply:
Fixed in latest (the scanning for forbidden RPATH entries was removed, but differences are noted at the top of the file).